pcdsdevices/attenuator.py

```diff
@@ -27,6 +28,31 @@
 logger = logging.getLogger(__name__)
 MAX_FILTERS = 12
 
+class Gatt_Y_aperture(TwinCATStatePMPS):
+    """
+    Controls the Y axis component of the apertures inside AT1K0
+    Defines the state count as 4 (5.5mm, 8mm, 10mm, 13mm) to limit the number of
+    config PVs we connect to.
+    """
+
+    states_list = []
```

Comment:
You've overridden the default values of `in_states` and `out_states`, but aren't `state_list` and `_in_if_not_out` the same as defined in `TwinCATStatePMPS`?  

I'm maybe not well informed about the device we're building, but how does this work?  If there's no in or out state, is `TwinCATStatePMPS` (a subclass of an `InOutPositioner`) the right base class for this?

Reply:
I think you may be right that the states lists are unnecessary, @tangkong. Strictly speaking the only thing that should be required here is the `UpCpt`, I believe, but @ZLLentz can correct me if I'm wrong (the machinery is complicated here!).
 
The goal is to mainly change the number of states in the `config` component from the statically-defined default of 10 (if I recall correctly) because it can vary on a device-by-device basis. That is, even if there are PVs that exist for all 10 states, we only want the first 4 here for Y and the first one for X (below).

Reply:
You do need to clear most of the defaults to avoid errors like "you don't have an OUT state, what gives?" if your device doesn't have an "OUT" state, so I told Mike to clear them all, but it's likely that you're correct and some of these are not needed.

Reply:
yes looking back at a couple levels of inheritance, it looks like `states_list=[]` and and `_in_if_not_out=True` are already defined in [TwincatInOutPositioner](https://github.com/mkestra/pcdsdevices/blob/5ae8c2cb7846402001d2dd3593b26d4e1bee84f7/pcdsdevices/inout.py#L221l). I will remove these.